commit 631f0893038743cebd2def39df61aceb48bd43a9
Author: David du Colombier <0intro@gmail.com>
Date:   Sun Sep 13 23:40:43 2015 +0200

    wpa_supplicant: fix static link with readline
    
    The readline library depends on ncurses, so
    it should be set before ncurses on the linker
    command line to be able to be statically linked
    successfully.

    Signed-off-by: David du Colombier <0intro@gmail.com>

diff --git a/wpa_supplicant/Makefile b/wpa_supplicant/Makefile
index 1597412..581db02 100644
--- a/wpa_supplicant/Makefile
+++ b/wpa_supplicant/Makefile
@@ -1408,7 +1408,7 @@ LIBS += $(DBUS_LIBS)
 
 ifdef CONFIG_READLINE
 OBJS_c += ../src/utils/edit_readline.o
-LIBS_c += -lncurses -lreadline
+LIBS_c += -lreadline -lncurses
 else
 ifdef CONFIG_WPA_CLI_EDIT
 OBJS_c += ../src/utils/edit.o
